server/services/permissionsService.test.ts

```diff
@@ -0,0 +1,785 @@
+
+        expect(permissions.activity).toEqual({
+          view: false,
+          edit: false,
+        })
+      })
+
+      it('should return edit false if prisoner status is ACTIVE OUT', async () => {
+        const user = { ...prisonUserMock, userRoles: [Role.ActivityHub] }
+        const prisoner = { ...PrisonerMockDataA, status: 'ACTIVE OUT' }
+
+        const permissions = await service.getOverviewPermissions(user, prisoner)
+
+        expect(permissions.activity).toEqual({
+          view: false,
+          edit: false,
+        })
+      })
+    })
+
+    describe('pathfinder', () => {
+      it.each([
+        Role.PathfinderApproval,
+        Role.PathfinderStdPrison,
+        Role.PathfinderStdProbation,
+        Role.PathfinderHQ,
+        Role.PathfinderUser,
+        Role.PathfinderLocalReader,
+        Role.PathfinderNationalReader,
+        Role.PathfinderPolice,
+        Role.PathfinderPsychologist,
+      ])('should return view true if user has %s role', async role => {
+        const user = { ...prisonUserMock, userRoles: [role] }
+
+        const permissions = await service.getOverviewPermissions(user, PrisonerMockDataA)
+
+        expect(permissions.pathfinder.view).toEqual(true)
+      })
+
+      it.each([
+        Role.PathfinderApproval,
+        Role.PathfinderStdPrison,
+        Role.PathfinderStdProbation,
+        Role.PathfinderHQ,
+        Role.PathfinderUser,
+      ])('should return edit true if user has %s role', async role => {
+        const user = { ...prisonUserMock, userRoles: [role] }
+
+        const permissions = await service.getOverviewPermissions(user, PrisonerMockDataA)
+
+        expect(permissions.pathfinder.edit).toEqual(true)
+      })
+
+      it.each([
+        Role.PathfinderLocalReader,
+        Role.PathfinderNationalReader,
+        Role.PathfinderPolice,
+        Role.PathfinderPsychologist,
+      ])('should return view true, edit false if user has %s role', async role => {
+        const user = { ...prisonUserMock, userRoles: [role] }
+
+        const permissions = await service.getOverviewPermissions(user, PrisonerMockDataA)
+
+        expect(permissions.pathfinder).toEqual({
+          view: true,
+          edit: false,
+        })
+      })
+
+      it('should return view and edit false if user has no pathfinder roles', async () => {
+        const permissions = await service.getOverviewPermissions(prisonUserMock, PrisonerMockDataA)
+
+        expect(permissions.pathfinder).toEqual({
+          view: false,
+          edit: false,
+        })
+      })
+    })
+
+    describe('soc', () => {
+      it.each([Role.SocCommunity, Role.SocCustody])('should return view true if user has %s role', async role => {
+        const user = { ...prisonUserMock, userRoles: [role] }
+
+        const permissions = await service.getOverviewPermissions(user, PrisonerMockDataA)
+
+        expect(permissions.soc.view).toEqual(true)
+      })
+
+      // Odd rule, one to look into
```

Comment:
TODO raise with Joe Cartwright and team?

Reply:
Have raised with Joe Cartwright. As it is the same as what is currently there I don't think it is a reason to hold up this particular PR, but when I get a response I will update accordingly